app/views/editions/edition_thread/_assigned_to_event.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="event panel panel-default">
   <div class="panel-heading">
     <div class="pull-right">
-      <%= format_time(event.try(:updated_at) || event.try(:at)) %>
+      <%= format_time(event.edition.updated_at) %>
```

Comment:
If not `event.edition.updated_at.to_date.to_s(:rfc822)` then you can add a new format.. Doing it via internationalisation is the trendiness I think.

http://guides.rubyonrails.org/i18n.html#adding-date-time-formats